addons/zeusFactory/functions/fnc_ai_onRTB.sqf

```diff
@@ -0,0 +1,8 @@
+#include "script_component.hpp"
+
+params ["_leader"];
+private _vehicle = vehicle _leader;
+TRACE_2("ai_onRTB",_leader,_vehicle);
+
+deleteVehicle _leader;
+deleteVehicle _vehicle;
```

Comment:
done, btw checkout new `createGroup` in next patch:
https://community.bistudio.com/wiki/createGroup